--- a/scripts/bin/graphics-core22-provider-wrapper.in	2025-08-27 18:37:42.204138464 -0400
+++ b/scripts/bin/graphics-core22-provider-wrapper.in	2025-08-27 18:40:41.535366465 -0400
@@ -8,13 +8,13 @@
 for arch in ${ARCH_TRIPLETS[@]}; do
   GBM_BACKENDS_PATH=${GBM_BACKENDS_PATH:+$GBM_BACKENDS_PATH:}${SELF}/lib/${arch}/gbm
   LD_LIBRARY_PATH=${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}${SELF}/lib/${arch}:${SELF}/lib/${arch}/vdpau
-  LIBGL_DRIVERS_PATH=${LIBGL_DRIVERS_PATH:+$LIBGL_DRIVERS_PATH:}${SELF}/lib/${arch}/dri/
-  LIBVA_DRIVERS_PATH=${LIBVA_DRIVERS_PATH:+$LIBVA_DRIVERS_PATH:}${SELF}/lib/${arch}/dri/
+  LIBGL_DRIVERS_PATH=${SELF}/lib/jellyfin-ffmpeg/lib/dri:${LIBGL_DRIVERS_PATH:+$LIBGL_DRIVERS_PATH:}${SELF}/lib/${arch}/dri/
+  LIBVA_DRIVERS_PATH=${SELF}/lib/jellyfin-ffmpeg/lib/dri:${LIBVA_DRIVERS_PATH:+$LIBVA_DRIVERS_PATH:}${SELF}/lib/${arch}/dri/
 done
 __EGL_VENDOR_LIBRARY_DIRS=${__EGL_VENDOR_LIBRARY_DIRS:+$__EGL_VENDOR_LIBRARY_DIRS:}${SELF}/share/glvnd/egl_vendor.d
 __EGL_EXTERNAL_PLATFORM_CONFIG_DIRS=${__EGL_EXTERNAL_PLATFORM_CONFIG_DIRS:+$__EGL_EXTERNAL_PLATFORM_CONFIG_DIRS:}${SELF}/share/egl/egl_external_platform.d
-VK_LAYER_PATH=${VK_LAYER_PATH:+$VK_LAYER_PATH:}${SELF}/share/vulkan/implicit_layer.d/:${SELF}/share/vulkan/explicit_layer.d/
-XDG_DATA_DIRS=${XDG_DATA_DIRS:+$XDG_DATA_DIRS:}${SELF}/share
+VK_LAYER_PATH=${SELF}/lib/jellyfin-ffmpeg/share/vulkan/explicit_layer.d/:${VK_LAYER_PATH:+$VK_LAYER_PATH:}${SELF}/share/vulkan/implicit_layer.d/:${SELF}/share/vulkan/explicit_layer.d/
+XDG_DATA_DIRS=${SELF}/lib/jellyfin-ffmpeg/share:${XDG_DATA_DIRS:+$XDG_DATA_DIRS:}${SELF}/share
 
 # These are in the default LD_LIBRARY_PATH, but in case the snap dropped it inadvertently
 if [ -d "/var/lib/snapd/lib/gl" ] && [[ ! ${LD_LIBRARY_PATH} =~ (^|:)/var/lib/snapd/lib/gl(:|$) ]]; then
